apps/projection/src/app/ui/card/card.component.html

```diff
@@ -1,29 +1,30 @@
-<div class="border-2 border-black rounded-md p-4 w-fit flex flex-col gap-3" [class]="customClass">
+<div
+  class="border-2 border-black rounded-md p-4 w-fit flex flex-col gap-3"
+  [ngClass]="{
+    'bg-light-red': isTeacher,
+    'bg-light-green': isStudent
+  }">
   <img
     *ngIf="type === CardType.TEACHER"
     src="assets/img/teacher.png"
-    width="200px"
-  />
+    width="200px" />
   <img
     *ngIf="type === CardType.STUDENT"
     src="assets/img/student.webp"
-    width="200px"
-  />
+    width="200px" />
 
   <section>
-    <app-list-item
-      *ngFor="let item of list"
-      [name]="item.firstname"
-      [id]="item.id"
-      [type]="type"
-    >
-    </app-list-item>
+    <ng-container *ngFor="let item of list">
+      <ng-template
+        [ngTemplateOutlet]="cardList"
+        [ngTemplateOutletContext]="{ item }">
```

Comment:
you can use the $implicit property for the first value.

This way you will only use `let-xxx` inside your template instead of `let-xxx=item`